### PR TITLE
Integration obj documentation

### PR DESCRIPTION
**Pull request by vidhyakpai**
Updating the Readme for Integration Objects and for the parent.

### CODE REVIEW

File: packages/integration-objects/README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# SAP Cloud SDK Currency Conversion Integration Objects
+
+The SAP Cloud SDK currency conversion library is a TypeScript library that can be used to convert currency exchange rates. You can perform conversions with either one currency pair or multiple conversions through a single API call. You can use a data adapter to provide exchange rates, configuration settings, and so on.
+
+## Integration Objects
+
+Integration objects are reusable artifacts consisting of data models that send the required information to the library, and services that operate on the data models. These artifacts are shipped with Currency Conversion and can be used in addition to the library.
```
Comment (prilk) [inline packages/integration-objects/README.md:7]:
```suggestion
Integration objects are reusable artifacts consisting of data models that send the required information to the library and services that operate on the data models. These artifacts are shipped with Currency Conversion and can be used in addition to the library.
```

File: packages/integration-objects/README.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+Integration objects come with data models and services that you can use to send persisted information to the library, rather than providing all the information at run time. Integration objects are intended to help with the use of the library but you can consume the library without them by using a custom data adapter.
```
Comment (prilk) [inline packages/integration-objects/README.md:9]: The previous para and this one can be combined as most of the info is repeated.

File: packages/integration-objects/README.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+The OData services provided are as follows:
```
Comment (prilk) [inline packages/integration-objects/README.md:11]:
```suggestion
The following OData services are provided:
```

**Review by prilk — COMMENTED** [review]
@vidhyakpai: the review is complete.

File: packages/integration-objects/README.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
+# SAP Cloud SDK Currency Conversion Integration Objects
+
+The SAP Cloud SDK currency conversion library is a TypeScript library that can be used to convert currency exchange rates. You can perform conversions with either one currency pair or multiple conversions through a single API call. You can use a data adapter to provide exchange rates, configuration settings, and so on.
```
Comment (FrankEssenberger) [inline packages/integration-objects/README.md:3]:
```suggestion
The SAP Cloud SDK currency conversion library is a TypeScript library that can be used to convert amounts to different currencies. You can perform conversions with either one currency pair or multiple conversions through a single API call. You can use a data adapter to provide exchange rates, configuration settings, and so on.
```

**Review by FrankEssenberger — APPROVED** [review]
Once you handled the suggestions from @prilk please go ahead and merge.

File: packages/integration-objects/README.md (hunk 4 above)
Comment (vidhyakpai) [reply packages/integration-objects/README.md]: Done

File: packages/integration-objects/README.md (hunk 2 above)
Comment (vidhyakpai) [reply packages/integration-objects/README.md:9]: > The previous para and this one can be combined as most of the info is repeated.

@prilk , could you recommend how I can combine the two?

**Review by suhasraok — APPROVED** [review]

**Pull request merged**